certified-connectors/DocTOPDFConnector/Readme.md

```diff
@@ -0,0 +1,37 @@
+Word to PDF Converter
```

Comment:
Readme must follow this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) in order for PR to be accepted. Please refer to the template.